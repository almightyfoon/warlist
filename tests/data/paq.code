c2501b_-0O151r1X3q3k3k7j3E3r3r3s3C2_3G3H3G3H8M3O___-0d171U1U1U1U9x9x9x24243C3z3z3B3B2_2_
