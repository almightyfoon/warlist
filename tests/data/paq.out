https://conflictchamber.com/#c2501b_-0Om11r1Xme3k3k7j3E3r3r3s3C2_3G3H3G3H8M3O___-0d171U1U1U1U9x9x9x24243C3z3z3B3B2_2_

Protectorate Army - 73 / 75 points
[Theme] The Faithful Masses

[Harbinger 1] The Harbinger of Menoth [+27]
 - Devout [9]
 - Guardian [15]
 - Hierophant [3]
Allegiant of the Order of the Fist [3]
Allegiant of the Order of the Fist [3]
Anastasia di Bray [3]
High Paladin Dartan Vilmon [0(6)]
Paladin of the Order of the Wall [4]
Paladin of the Order of the Wall [4]
Reclaimer Gatekeeper [3]
The Covenant of Menoth [0(4)]
Choir of Menoth (min) [4]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [5]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [0(5)]
Lady Aiyana & Master Holt [8]
 - Attendant Priest [3]

Protectorate Army - 75 / 75 points
[Theme] The Creator's Might

[Amon 1] High Allegiant Amon Ad-Raza [+29]
 - Crusader [10]
 - Crusader [10]
 - Crusader [10]
 - Crusader [10]
 - Repenter [8]
 - Repenter [8]
 - Repenter [8]
 - Templar [15]
 - Templar [15]
The Covenant of Menoth [0(4)]
Vassal of Menoth [0(3)]
Vassal of Menoth [0(3)]
Wrack [1]
Wrack [1]
Choir of Menoth (min) [4]
Choir of Menoth (min) [4]
